include/fi_util.h

```diff
@@ -330,7 +330,7 @@ struct util_event {
 	uint8_t			data[0];
 };
 
-int fi_eq_create(struct fid_fabric *fabric, struct fi_eq_attr *attr,
+int ofi_eq_create(struct fid_fabric *fabric, struct fi_eq_attr *attr,
 		 struct fid_eq **eq_fid, void *context);
```

Comment:
Thanks!  psm2 provider was using these and I missed that. I have updated the PR now.